docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -38,95 +77,117 @@ As for the domain URL, Feel free to get creative — just remember that the URL
+### 2.1 Enabling enhanced health checks
 
-You can now go ahead and click `Review and launch`.
+You will need to enable enhanced health checks for your Elastic Beanstalk environment as this will tell the load balancer that the environment is healthy.
 
-#### Enabling enhanced health checks
+Click on the `Edit` link under the Load Balancer section as seen here:
 
-You will need to enable enhanced health checks for your Beanstalk Monitoring.
+![Elastic Beanstalk Monitoring](images/EBLoadBalancerEdit.png)
 
-Click on the `Modify` link under the Monitoring section as seen here:
+In the Processes section, select the default process and click on `Actions` -> Edit
 
-![Elastic Beanstalk Monitoring](images/EBMonitoringSelect.png)
+![Elastic Beanstalk Monitoring Process](images/EBProcessesSection.png)
 
-The `Health check path` is how Elastic Beanstalk knows when the application is ready to run. This path is also used by the Load Balancer to check application heath. Set this path to `/api/health`.
+The `Health check path` is where the Load balancer asks the application if its healthy so it can send traffic to. Set this path to `/api/health`
 
-Then make sure enhanced health checks are enabled. This is a free option, unless you later add specific metrics to CloudWatch.
+![Elastic Beanstalk Monitoring endpoint](images/EBProcessEditEndpointHealth.png)
 
-![Elastic Beanstalk Monitoring Settings](images/EBMonitoringSettings.png)
+After configuring this health check you can click on `Save` at the bottom of the page
 
-#### Configuring RDS for Metabase
+### 2.2 Enabling VPC
 
-To run Metabase in a cloud environment of any kind we highly recommend using an independent database server with high availability such as Amazon RDS. So for standard deployments we will choose to create an RDS instance with our Elastic Beanstalk application.
+A Virtual Private Cloud (VPC) is a virtual network that will be logically isolated for the resources that you put inside. Inside these VPC's you can create subnets, firewall rules, route tables and many more. It's one of the foundational features of AWS and you can learn more about it [here](https://aws.amazon.com/vpc/faqs/)
 
-_NOTE: It's possible to skip this step if you wish. However, this will force Metabase to use a local H2 database file on your application server, and there will be no way to backup and maintain that database. **When your instance is restarted for any reason you'll lose all your Metabase data**. If you are just doing a quick trial of Metabase that may be okay, but otherwise we recommend against it._
+You must enable your Application to exist in a VPC, otherwise you will receive an error when creating it. To use a VPC, head to the Network section in the configuration and click on the `Edit` button.
```

Comment:
```suggestion
You must configure your Application to launch in a VPC, otherwise you will receive an error when creating it. To use a VPC, head to the Network section in the configuration and click on the `Edit` button.
```

> ... otherwise you will receive an error when creating it.

This doesn't seem accurate. What error will I receive, and why? Is a VPC a requirement? If so, rather say that.